Reddit/Reddit_Search.ipynb

```diff
@@ -0,0 +1,727 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "42fce9ae-03b7-467b-b5ba-4fd66ffc50ec",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "<img width=\"10%\" alt=\"Naas\" src=\"https://landen.imgix.net/jtci2pxwjczr/assets/5ice39g4.png?w=160\"/>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9861bc61-3c92-41a8-8489-aba0d2196fb6",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "# Reddit - Search"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "89c7636e-14c1-45ab-8cbc-d6826f469f6d",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**Tags:** #reddit #search #api #dev #get #python"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1138db6c-791e-4ae3-9d34-7038d3a97482",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**Author:** [Firstname Lastname](https://www.linkedin.com/in/xxxx)"
```

Comment:
@Dr0p42 You can add your LinkedIn Url

Reply:
Just updated it! Thank you @Dr0p42 for you contribution! Thank you @srini047 for the good catch :)